src/api/accounts/businesses.md

```diff
@@ -23,20 +23,28 @@ A Business is associated with a single [Account][].
 
 ### Business
 
-|     Field     |        Type        |                    Description                     |
-| :------------ | :----------------- | :------------------------------------------------- |
-| id            | String             | The unique identifier.                             |
-| accountId     | String             | The Centrapay accountId for an org account.        |
-| accountName   | String             | The Centrapay account name for an org account.     |
-| test          | Boolean            | A flag which is present if the [Account][] is test |
-| nzbn          | String             | The unique NZBN identifier.                        |
-| name          | String             | Legal name recorded in the Companies Register.     |
-| tradingName   | String             | Trading name recorded in the Companies Register.   |
-| companyNumber | String             | Company number recorded in the Companies Register. |
-| createdAt     | {% dt Timestamp %} | When the Business was created.                     |
-| updatedAt     | {% dt Timestamp %} | When the Business was updated.                     |
-| createdBy     | {% dt CRN %}       | The User or API Key that created the Business.     |
-| updatedBy     | {% dt CRN %}       | The User or API Key that updated the Business.     |
+|     Field     |                Type                 |                     Description                     |
+| :------------ | :---------------------------------- | :-------------------------------------------------- |
+| id            | String                              | The unique identifier.                              |
+| accountId     | String                              | The Centrapay accountId for an org account.         |
+| accountName   | String                              | The Centrapay account name for an org account.      |
+| test          | Boolean                             | A flag which is present if the [Account][] is test  |
+| nzbn          | String                              | The unique NZBN identifier.                         |
+| name          | String                              | Legal name recorded in the Companies Register.      |
+| tradingName   | String                              | Trading name recorded in the Companies Register.    |
+| companyNumber | String                              | Company number recorded in the Companies Register.  |
+| createdAt     | {% dt Timestamp %}                  | When the Business was created.                      |
+| updatedAt     | {% dt Timestamp %}                  | When the Business was updated.                      |
+| createdBy     | {% dt CRN %}                        | The User or API Key that created the Business.      |
+| updatedBy     | {% dt CRN %}                        | The User or API Key that updated the Business.      |
+| taxConfig     | [Tax Config](#tax-config) {% opt %} | The value-added tax configuration for the Business. |
```

Comment:
What does `value-added` mean? Can we just remove it?

Reply:
It is the specific type of tax. https://en.wikipedia.org/wiki/Value-added_tax